src/components/data-model-table/association-selection.tsx

```diff
@@ -131,173 +149,113 @@ export const AssociationSelection = <
     setSorters,
     setCurrent,
     setPageSize,
-  } = useTable<GetQuery>({
-    ...tableOptions,
-  });
+  } = useTable<GetQuery>(tableOptions);
 
   const [selectedRows, setSelectedRows] = useState<AssociatedModel[]>(() =>
     value ? [value] : [],
   );
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
good to see that this section of code is no longer needed and we can plainToInstance elsewhere like in the reducer 👍